app/v1/controllers/requestParsers/validators/SubmitReturnValidator.scala

```diff
@@ -0,0 +1,143 @@
+      VATNetValueValidation.validate(body.totalVatDue, body.vatReclaimedCurrPeriod, body.netVatDue),
+      VATTotalValueValidation.validate(body.vatDueSales, body.vatDueAcquisitions, body.totalVatDue)
+    )
+  }
+
+  private def vrnFormatValidation: SubmitRawData => List[List[MtdError]] = (data: SubmitRawData) => {
+    List(
+      VrnValidation.validate(data.vrn)
+    )
+  }
+
+  private def monetaryFormatValidation: SubmitRawData => List[List[MtdError]] = (data: SubmitRawData) => {
+    val body: SubmitRequestBody = data.body.asJson.get.as[SubmitRequestBody]
+
+    List(
+      MonetaryValueFormatValidation.validate(body.vatDueSales, "vatDueSales"),
+      MonetaryValueFormatValidation.validate(body.vatDueAcquisitions, "vatDueAcquisitions"),
+      MonetaryValueFormatValidation.validate(body.totalVatDue, "totalVatDue"),
+      MonetaryValueFormatValidation.validate(body.vatReclaimedCurrPeriod, "vatReclaimedCurrPeriod"),
+      MonetaryValueFormatValidation.validate(body.netVatDue, "netVatDue"),
+      MonetaryValueFormatValidation.validate(body.totalValueSalesExVAT, "totalValueSalesExVAT"),
+      MonetaryValueFormatValidation.validate(body.totalValuePurchasesExVAT, "totalValuePurchasesExVAT"),
+      MonetaryValueFormatValidation.validate(body.totalValueGoodsSuppliedExVAT, "totalValueGoodsSuppliedExVAT"),
+      MonetaryValueFormatValidation.validate(body.totalAcquisitionsExVAT, "totalAcquisitionsExVAT"),
+    )
+  }
+
+  private def monetaryValueValidation: SubmitRawData => List[List[MtdError]] = (data: SubmitRawData) => {
+    val minNetVatDue = BigDecimal(0.00)
+    val maxNetVatDue = BigDecimal(99999999999.99)
+    val minRegularValue = BigDecimal(-9999999999999.99)
+    val maxRegularValue = BigDecimal(9999999999999.99)
+
+    val body: SubmitRequestBody = data.body.asJson.get.as[SubmitRequestBody]
+
+    List(
+      MonetaryValueRangeValidation.validate(body.vatDueSales, "vatDueSales", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.vatDueAcquisitions, "vatDueAcquisitions", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.totalVatDue, "totalVatDue", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.vatReclaimedCurrPeriod, "vatReclaimedCurrPeriod", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.netVatDue, "netVatDue", minNetVatDue, maxNetVatDue),
+      MonetaryValueRangeValidation.validate(body.totalValueSalesExVAT, "totalValueSalesExVAT", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.totalValuePurchasesExVAT, "totalValuePurchasesExVAT", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.totalValueGoodsSuppliedExVAT, "totalValueGoodsSuppliedExVAT", minRegularValue, maxRegularValue),
+      MonetaryValueRangeValidation.validate(body.totalAcquisitionsExVAT, "totalAcquisitionsExVAT", minRegularValue, maxRegularValue),
```

Comment:
The error messages for **totalValueSalesExVAT**, **totalValuePurchasesExVAT**, **totalValueGoodsSuppliedExVAT** and **totalAcquisitionsExVAT** are different ---> **The value must be between -9999999999999 and 9999999999999**